src/pages/vendors/components/AdditionalInfo.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Card, Element } from '@invoiceninja/cards';
+import { SelectField, Textarea } from '@invoiceninja/forms';
+import { useStaticsQuery } from 'common/queries/statics';
+import { Alert } from 'components/Alert';
+import { useTranslation } from 'react-i18next';
+
+type Props = { data?: any; formik?: any; errors?: any };
+
+export function AdditionalInfo(props: Props) {
+  const [t] = useTranslation();
+  const { data: statics } = useStaticsQuery();
+
+  return (
+    <Card title={t('additional_info')} className="mb-5">
+      {props.errors?.notes && <Alert type="danger">{props.errors.notes}</Alert>}
+      <Element leftSide={t('currency')}>
+        <SelectField
+          onChange={(event: any) => {
+            props.formik.setFieldValue('currency_id', event.target.value);
+          }}
+        >
+          {statics?.data.currencies.map((element: any, index: any) => {
+            if (element.id === props.data.currency_id) {
```

Comment:
Is this way we preselect the default? If so, this is not the way. All you have to do is provide `defaultValue` to the select component with an ID.